### PR TITLE
Navigation and avatar changes

### PR DESCRIPTION
**Pull request by arivepr**
This PR is to move in the changes made for cards 3085 and 8493. 
The PF Avatar is now all integrated with the dropdown menu and "Documentation" link has been moved to the bottom of the nav.

### CODE REVIEW

File: src/js/App/Sidenav/Navigation.js
Comment (ryelo) [inline src/js/App/Sidenav/Navigation.js:21]: Extra space

File: src/js/App/Sidenav/Navigation.js
Comment (ryelo) [inline src/js/App/Sidenav/Navigation.js:28]: Extra space

File: src/js/App/Header/UserToggle.js
Comment (ryelo) [inline src/js/App/Header/UserToggle.js:85]: Move this back up to line 84

File: src/sass/chrome.scss
Comment (ryelo) [inline src/sass/chrome.scss:102]: Can you move this to a new file `UserIcon.scss` and import it in `UserIcon.js`?

File: src/sass/chrome.scss
Comment (ryelo) [inline src/sass/chrome.scss:114]: extra space

File: src/js/App/Header/UserIcon.js
Comment (ryelo) [inline src/js/App/Header/UserIcon.js:29]: Extra space

**Review by ryelo — DISMISSED** [review]

**Comment by ryelo** [comment]
@arivepr You can run `npm run lint` to generate these errors, but copying the lint:
```
/home/travis/build/RedHatInsights/insights-chrome/src/js/App/Header/Tools.js

  14:8  error  'UserIcon' is defined but never used  no-unused-vars

/home/travis/build/RedHatInsights/insights-chrome/src/js/App/Header/UserToggle.js

  8:10  error  'Avatar' is defined but never used  no-unused-vars
```

File: src/js/App/Header/Tools.js
Comment (karelhala) [inline src/js/App/Header/Tools.js:2]: Extra character
```suggestion
import { Button } from '@patternfly/react-core/dist/js/components/Button/Button';
```

**Comment by karelhala** [comment]
Remove the extra ` and update snapshots and we are golden!

**Review by karelhala — APPROVED** [review]

**Comment by ryelo** [comment]
@karelhala sorry I added the `on hold` label because we were tweaking some loading states. We'll throw it in another branch 😄

**Comment by karelhala** [comment]
@ryelo too fast on the merge button 😁

**Pull request merged**
